README.md

```diff
@@ -47,7 +47,7 @@ jobs:
 | `run-tests` | Enable/Disable running tests  | Optional | `false` |
 | `branch` | The default flatpak branch  | Optional | `master` |
 | `cache` | Enable/Disable caching `.flatpak-builder` directory | Optional | `true` |
-| `cache-key` | Specifies the cache key | Optional | `flatpak-builder-${sha256(manifestPath)}` |
+| `cache-key` | Specifies the cache key. CPU arch is automatically added, so there is no need to add it to the cache key. | Optional | `flatpak-builder-${sha256(manifestPath)}` |
```

Comment:
Maybe modify the default value as well?

Reply:
The arch is added when trying to restore the cache, the default value still `flatpak-builder-${sha256(manifestPath)}`.

I think adding it to the default value is not a good idea because it goes against what the note explain.

Maybe adding under the table something more detailed like:

With `flatpak-builder-${sha256(manifestPath)}` as cache key option, `flatpak-builder-${sha256(manifestPath)}-x86_64` will be used for x86_64 caching and `flatpak-builder-${sha256(manifestPath)}-aarch64` for aarch64.

Reply:
The whole cache-key thing should ideally be private but it's public due an issue from GitHub actions side. 

The default value in the table is meant to be an easy copy/paste thing in case you want to modify the default behaviour.

Reply:
And yes, mentioning the possible values or explicitly saying that cache keys has to be different for each architecture would be fine as well